src/filters/truncate.js

```diff
@@ -22,7 +22,7 @@ angular.module('openshiftCommonUI')
         // Find the last word break, but don't look more than 10 characters back.
         // Make sure we show at least the first 5 characters.
         var startIndex = Math.max(4, charLimit - 10);
-        var lastSpace = truncated.lastIndexOf(/\s/, startIndex);
+        var lastSpace = truncated.lastIndexOf(' ', startIndex);
```

Comment:
@spadgett we've got a couple of options for handling this. We could completely ignore the word boundary flag if the limit is already met, like so...

```javascript
if (useWordBoundary !== false && truncated.length > charLimit) {
```
Or we could still try to include the word boundary then apply a second condition for removing the 10 characters if the length still isn't below/equal to the ```charLimit```. You have preference?